examples/mobile/ios/mobile/AppDelegate.h

```diff
@@ -0,0 +1,6 @@
+#import <RCTAppDelegate.h>
```

Comment:
This is because the default react native app creates also some code for IOS. No work has been done for libwaku on ios yet, but I think it's safe to leave this example code for now, since it will be the base for the IOS integration, even if we do not support it yet.

(Also because I fear breaking react native if this code does not exist :) )